packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterApiError.kt

```diff
@@ -20,20 +20,24 @@ import android.os.Looper
 import androidx.annotation.NonNull
 import com.amplifyframework.api.ApiException
 import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.MethodChannel.Result
+
 
 class FlutterApiError {
     companion object {
         private val handler = Handler(Looper.getMainLooper())
 
+        @JvmStatic
+        fun postFlutterError(flutterResult: Result, msg: String, errorMap: Map<String, Any>) {
+            handler.post { flutterResult.error("AmplifyException", msg, errorMap) }
+        }
```

Comment:
Is this needed or can we use the `postFlutterError()` method defined below?

Reply:
It's needed because the kotlin methods require an actual exception object.

Reply:
As discussed offline we can create the Exception for that case and pass it in